spec/support/facility_statements_shared_examples.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.shared_examples "it sets up order_detail and creates statements" do
```

Comment:
This is more of a personal preference, but I prefer to leave the shared examples in the spec file if it's only used in a single file. One less file to track down.